lib/plugins/aws/package/compile/functions/index.js

```diff
@@ -719,6 +746,48 @@ class AwsCompileFunctions {
       Value: 'Value',
     };
   }
+
+  addFileContentsToHashes(filePath, hashes) {
```

Comment:
I think those two _methods_ are not conceptually methods, but rather local utils. I would put them as regular functions in module body (no need to expose them on class)